pkg/networkplugin-syncer/handlers/ovn/handler.go

```diff
@@ -27,7 +27,8 @@ import (
 	"github.com/submariner-io/submariner/pkg/networkplugin-syncer/handlers/ovn/nbctl"
 )
 
-var WaitingForLocalEndpoint = errors.New("Waiting for the local endpoint details before we can " +
+//nolint:stylecheck // Limit line length as per lll
+var ErrWaitingForLocalEndpoint = errors.New("Waiting for the local endpoint details before we can " +
 	"setup any remote endpoint related information, this will be retried.")
```

Comment:
```suggestion
var ErrWaitingForLocalEndpoint = errors.New("waiting for the local endpoint details before we can " +
	"setup any remote endpoint related information, this will be retried")
```